### PR TITLE
Minor improvements for retail best practices

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Minor improvements following https://github.com/uyuni-project/uyuni-docs/pull/2277

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3 https://github.com/uyuni-project/uyuni-docs/pull/2393
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/retail/pages/retail-best-practices.adoc [hunk 1]
```diff
@@ -1,75 +1,64 @@
...
+== Deploying new images
 
 This file describes how to deploy a new image on selected terminals first, test it, and then deploy it to all terminals.
 
-The hardware type (HWTYPE) group is used for booting new terminals, while already deployed terminals can
-be moved to any group that provides the {saltboot} formula. 
-This flexibility allows for testing of new images
-or configurations. 
-To ensure a smooth deployment, follow these steps:
+The hardware type group [literal]``HWTYPE`` is used for booting new terminals, while already deployed terminals can be moved to any group that provides the {saltboot} formula.
```
Comment (aaannz) [inline modules/retail/pages/retail-best-practices.adoc:8]: This is sending different message then originally thought.
The intent was to provide some recognizable string for hardware type group.

In the new version it seems like we are talking about hardware type group named `HWTYPE` which is not the case. `HWTYPE` is just a prefix for the hardware type group name.

File: modules/retail/pages/retail-best-practices.adoc [hunk 2]
```diff
@@ -1,75 +1,64 @@
...
 
+. Build new image.
+    Use image synchronization formula to synchronize the image to all Branch Servers.
+. Create a test group.
   * Name the group, for example, [literal]``TEST:POSVendor-Terminal``. The name can be chosen freely.
-  * Add the {saltboot} formula to this group, replicating the configuration from the original HWTYPE group, except for the image version. Use the name and version of the new image.
-
-* Move Selected Terminals
-
+  * Add the {saltboot} formula to this group, replicating the configuration from the original [literal]``HWTYPE`` group, except for the image version.
```
Comment (aaannz) [inline modules/retail/pages/retail-best-practices.adoc:18]: Maybe I am reading it wrong, but I would just write `from the original hardware type group`.

File: modules/retail/pages/retail-best-practices.adoc [hunk 3]
```diff
@@ -1,75 +1,64 @@
...
+The hardware type group [literal]``HWTYPE`` is used for booting new terminals, while already deployed terminals can be moved to any group that provides the {saltboot} formula. 
+This flexibility allows for testing of new images or configurations. 
 
-* Build New Image
-
-  * Use Image Sync formula to synchronize the image to all Branch Servers
-
-* Create a Test Group
+//To ensure a smooth deployment, follow these steps:
+.Procedure: Smooth image deployment
```
Comment (aaannz) [inline modules/retail/pages/retail-best-practices.adoc:12]: Smooth is interesting choice 😃 

This procedure is how one can have two groups controlling the image deployment. In practice this is mimicking A/B image deployment. So we can call this either `A/B image deployment`, or `Dual group image deployment`.

File: modules/retail/pages/retail-best-practices.adoc [hunk 4]
```diff
@@ -1,75 +1,64 @@
...
 
 By default, {saltboot} formula selects the specified image name and version or selects the highest one if the version is not specified.
 
-There are multiple methods to control the image version within the HWTYPE and TEST groups:
+There are multiple methods to control the image version within the [literal]``HWTYPE`` and [literal]``TEST``` groups:
 
-* Specify Exact Version
-
-  * In both the HWTYPE and TEST groups, specify explicitly the exact version of the image to be deployed.
+.Procedure: Image control
```
Comment (aaannz) [inline modules/retail/pages/retail-best-practices.adoc:42]: This paragraph is not a procedure, but rather list of options how can one control image deployment.
* specifying exact versions in A/B group created above
* using inactive flag to deactivate image
* using freeze flag to disable image autoupdate

File: modules/retail/pages/retail-best-practices.adoc (hunk 3 above)
Comment (0rnela) [inline modules/retail/pages/retail-best-practices.adoc:12]:
```suggestion
.Procedure: Dual group image deployment
```

File: modules/retail/pages/retail-best-practices.adoc (hunk 2 above)
Comment (0rnela) [inline modules/retail/pages/retail-best-practices.adoc:18]:
```suggestion
  * Add the {saltboot} formula to this group from the original hardware type group. 
```

File: modules/retail/pages/retail-best-practices.adoc (hunk 1 above)
Comment (0rnela) [inline modules/retail/pages/retail-best-practices.adoc:8]:
```suggestion
The hardware type group (e.g [literal]``HWTYPE``) is used for booting new terminals, while already deployed terminals can be moved to any group that provides the {saltboot} formula. 
```

File: modules/retail/pages/retail-best-practices.adoc [hunk 5]
```diff
@@ -5,17 +5,17 @@
 
 This file describes how to deploy a new image on selected terminals first, test it, and then deploy it to all terminals.
 
-The hardware type group [literal]``HWTYPE`` is used for booting new terminals, while already deployed terminals can be moved to any group that provides the {saltboot} formula. 
+The hardware type group (e.g [literal]``HWTYPE``) is used for booting new terminals, while already deployed terminals can be moved to any group that provides the {saltboot} formula.
```
Comment (aaannz) [inline modules/retail/pages/retail-best-practices.adoc:8]:
```suggestion
The hardware type group (i.e. [literal]``HWTYPE`` group) is used for booting new terminals, while already deployed terminals can be moved to any group that provides the {saltboot} formula. 
```

File: modules/retail/pages/retail-best-practices.adoc [hunk 6]
```diff
@@ -1,75 +1,64 @@
...
 
+. Build new image.
+    Use image synchronization formula to synchronize the image to all Branch Servers.
+. Create a test group.
   * Name the group, for example, [literal]``TEST:POSVendor-Terminal``. The name can be chosen freely.
-  * Add the {saltboot} formula to this group, replicating the configuration from the original HWTYPE group, except for the image version. Use the name and version of the new image.
-
-* Move Selected Terminals
-
+  * Add the {saltboot} formula to this group from the original hardware type group.
```
Comment (nadvornik) [inline modules/retail/pages/retail-best-practices.adoc:18]: I am not sure if this is clear enough.
They have to manually enter the same values again.

Comment (0rnela) [reply modules/retail/pages/retail-best-practices.adoc:18]: @nadvornik -- can you please add text that would correctly reflect that?

Comment (nadvornik) [reply modules/retail/pages/retail-best-practices.adoc:18]: Something like: Add the {saltboot} formula to this group. Use the same configuration as in the original HWTYPE group, except for the image version. Use the name and version of the new image.

File: modules/retail/pages/retail-best-practices.adoc [hunk 7]
```diff
@@ -1,75 +1,64 @@
...
+//.Procedure: Image control
+* Specify exact version.
+** In both the [literal]``HWTYPE`` and [literal]``TEST`` groups, specify explicitly the exact version of the image to be deployed.
+** Make sure that the specified image version is already synchronized.
+* Mark new image as [literal]``Inactive``.
+** In the [literal]``HWTYPE`` group, do not specify the image version.
+** Mark the new image as [literal]``Inactive`` after building and before synchronizing it to Branch Server.
+** The [literal]``inactive`` flag ensures, that the image is not auto-selected when the image version is not specified.
+** After testing, remove the [literal]``Inactive`` flag so the image can be auto-selected in the [literal]``HWTYPE`` group.
+** The [literal]``Inactive`` flag can be accessed through the API. i
```
[the remark this answers is not in the document]
Comment (0rnela) [reply modules/retail/pages/retail-best-practices.adoc]: thank you

**Review by nadvornik — APPROVED** [review]
LGTM

**Review by aaannz — APPROVED** [review]

**Pull request merged**